### ISSUE TITLE
FQL no longer works.

### ISSUE THREAD

**Issue by phiresky** [phiresky/fbstats#6]
Facebook deprecated the FQL Api in version v2.0 and removed it completely in v2.1. This means this no longer works if the "app" is created after Aug 2014.

The newer Api limits the message loading to 25 messages per request (20x slower) and blocks after very few calls, making it almost impossible to get >10k messages.

If anybody has an idea to fix this, feel free to comment.

**Comment by phiresky** [comment]
A possible solution would be to be able to import the  `messages.html` from facebook export..

**Comment by xjimdim** [comment]
yeah please do that!! i downloaded my facebook export but I cant access the html file because its HUGE (100mb), Chrome crashes :P

**Comment by phiresky** [comment]
Well this tool doesn't allow actually reading the contents so it wouldn't really help with that ;)

**Comment by gadour97** [comment]
any news?

**Comment by phiresky** [comment]
I think they now also allow JSON export, which is even easier to parse. Only problem is the file size so it probably needs a streaming JSON parser. Shouldn't be hard, you can use FileReader API to get a file from file system in chunks and then parse in those chunks. But I don't use Facebook anymore so I don't really have a large incentive to implement this. PRs welcome of course.
